numismatic/cli.py

```diff
@@ -274,8 +274,10 @@ def run(state, timeout):
         timeout = None
     loop = asyncio.get_event_loop()
     logger.debug('starting ...')
-    tasks = {name:asyncio.Task(sub.listener) for name, sub in
-             state['subscriptions'].items() if sub.listener is not None}
+    # We have one _listener() per client
+    # tasks = {id(sub.client):asyncio.Task(sub.client.listener) 
+    #          for name, sub in state['subscriptions'].items()}
+    tasks = {}
```

Comment:
The _listener()-s are now started at instance creation time with asyncio.ensure_future() so this is not necessary anymore. I'm thinking about how to remove the whole `run` command as @barrysteyn  requested.